### ISSUE TITLE
Data input format

### ISSUE THREAD

**Issue by adithyaxx** [AIZOOTech/FaceMaskDetection#94]
Hi,

This is a great model and I am trying to retrain it with my own dataset in tensorflow keras. Could you share the loss functions and data input format you used?
